Helper/Multishipping.php

```diff
@@ -83,19 +99,14 @@ public function getQuote($quoteId = null): \Magento\Quote\Api\Data\CartInterface
         $quote = $this->getSessionQuote();
 
         // API Request
-        if (empty($quote) || !is_numeric($quote->getGrandTotal()))
-        {
-            try
-            {
+        if (empty($quote) || !is_numeric($quote->getGrandTotal())) {
+            try {
                 if ($quoteId)
                     $quote = $this->quoteRepository->get($quoteId);
                 else if ($this->quoteId) {
                     $quote = $this->quoteRepository->get($this->quoteId);
                 }
-            }
-            catch (\Exception $e)
-            {
-
+            } catch (\Exception $e) {
```

Comment:
This doesn't look right, we're just suppressing the exception. It looks like we could end up returning invalid data.

- [ ] Validate exception handling for Multishipping Helper

Reply:
agreed, will look into refactoring this to be better